examples/persistent-storage/KeyValueStorageTest.cpp

```diff
@@ -36,7 +36,8 @@ using namespace chip::DeviceLayer::PersistedStorage;
         {                                                                                                                          \
             char error_str[255];                                                                                                   \
             chip::FormatCHIPError(error_str, sizeof(error_str), temp_test_result);                                                 \
-            ChipLogError(NotSpecified, "%s: FAILED %ld [%s]", #test_result, temp_test_result, chip::ErrorStr(temp_test_result));   \
+            ChipLogError(NotSpecified, "%s: FAILED %ld [%s]", #test_result, static_cast<long int>(temp_test_result),               \
```

Comment:
I think most places we use `PRId32` for CHIP_ERROR.

Though maybe we should just add a CHIP_CONFIG_* define for the right format to use for it and start using that?  Similar to how `BLE_CONFIG_ERROR_FORMAT` works.

Reply:
Fixed